### ISSUE TITLE
Out of date section "what about kernels and bootloaders?"

### ISSUE THREAD

**Issue by xnox** [canonical/ubuntu.com#11595]
Out of date section "what about kernels and bootloaders?"

It mentions that one needs certificate in a different format (DER), but that is no longer sufficient.

The certificate needs to be different too, without the kernel-module signing EKU set which modern shims reject.

How can I submit revision for this blog post?

---
*Reported from: https://ubuntu.com/blog/how-to-sign-things-for-secure-boot*

**Comment by pmahnke** [comment]
do you want edit rights and you can update it directly in wordpress?

**Comment by davegoddard42** [comment]
@pmahnke @xnox - what is the status of this? Thanks

**Comment by xnox** [comment]
> do you want edit rights and you can update it directly in wordpress?

I would like the wordpress text "source" copy yes please, then I can propose an edit. I do not want to have edit powers or login on wordpress, sounds very dangerous. I am not good with wordpress.

**Comment by pmahnke** [comment]
@xnox I created a google doc and shared it with you.

**Comment by davegoddard42** [comment]
@carlos are you taking a look at this?

**Comment by mtruj013** [comment]
@pmahnke @xnox are we free to close this?

**Comment by xnox** [comment]
> @pmahnke @xnox are we free to close this?

I don't think so.... as I haven't yet provided the update to the blog post, and the update has not yet been published. Meaning the blog post still contains out of date / incorrect content.

**Comment by mtruj013** [comment]
I've reached out to marketing as both OP and the blog author have left the company. See [here](https://chat.canonical.com/canonical/pl/7u8kuy5a73fjfmkr9dcbrdydtr)

**Comment by muhammad-ali-pk** [comment]
I have reached out to the marketing to remind them of this pending update.

**Comment by Onibenjo** [comment]
I have reached out to the marketing team to remind them of this update.
